pkg/helpers/tokencmd/negotiator_gssapi.go

```diff
@@ -1,197 +1,77 @@
-// +build gssapi
```

Comment:
Does this work on windows, then? You couldn't use `/etc/krb5.conf` ever as the default location, then. What happens when both this handler and `SSPI` are active?

Reply:
I'd expect it to, yes. With this it won't be required to run `kinit` beforehand to get the ticket.
I think it might be easiest if we require the user to set the path to the conf file as an env var and read it from there, which should work on all OSes.